.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
 *.log
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.idea
+dist
```

Comment:
@IgorNadj Do you know if there's a reason for committing the build directory (`dist`)? Was it accidentally left out of the `.gitigore`?

Reply:
No reason at all, just consistently done like that. There's a card here about it to go through all our packages https://trello.com/c/ctKm2TTz/22-change-our-js-packages-bundle-src-not-dist

Reply:
I think you actually have to commit the build artifacts as well otherwise the consuming apps won't have the changes. Not sure if they are built on deployment of the consuming app.

Reply:
hmmm maybe not